wiki/en/Command-Line-Options.md

```diff
@@ -1,68 +0,0 @@
----
```

Comment:
Doesn't this also need a redirect?

Reply:
Where to? :-) It now exists in two places.

Reply:
Hmm. That's problematic then. Server manual?

Reply:
OK sure.